fs/accounting/stats.go

```diff
@@ -37,12 +39,130 @@ type StatsInfo struct {
 
 // NewStats cretates an initialised StatsInfo
 func NewStats() *StatsInfo {
-	return &StatsInfo{
+	s := &StatsInfo{
 		checking:     newStringSet(fs.Config.Checkers),
 		transferring: newStringSet(fs.Config.Transfers),
 		start:        time.Now(),
 		inProgress:   newInProgress(),
 	}
+
+	rc.Add(rc.Call{
+		Path:  "stats/transfers",
+		Fn:    s.bwStats,
+		Title: "Get transfer stats",
+		Help: `
+Show statistics for all transfers.
+`,
+	})
+
+	rc.Add(rc.Call{
+		Path:  "stats/reset",
+		Fn:    s.bwResetStats,
+		Title: "Reset transfer stats",
+		Help: `
+Resets the global transfer statistic counters.
+`,
+	})
+
+	return s
+}
+
+// reset statistics as requested by rc stats/reset
+func (s *StatsInfo) bwResetStats(in rc.Params) (out rc.Params, err error) {
+	out = make(rc.Params)
+
+	s.mu.Lock()
+	s.bytes = 0
+	s.errors = 0
+	s.checks = 0
+	s.transfers = 0
+	s.deletes = 0
+	s.start = time.Now()
+	s.mu.Unlock()
+
+	out["status"] = "ok"
+	out["message"] = "Global transfer statistic counters were reset to Zero"
+	return out, nil
+}
+
+// create map with statistics for rc stats/transfers
+func (s *StatsInfo) bwStats(in rc.Params) (out rc.Params, err error) {
+	out = make(rc.Params)
+	g, t, c, err := s.Map()
+	if err != nil {
+		return out, errors.Errorf("error while getting bw stats")
+	}
+	out["status"] = "ok"
+	out["global"] = g
+	out["transferring"] = t
+	out["checking"] = c
+	return out, nil
+}
+
+// Map convert the StatsInfo to a map for rc stats
+func (s *StatsInfo) Map() (map[string]interface{}, map[string]map[string]interface{}, map[string]map[string]interface{}, error) {
+	dataRateMultiplier := 1.0
```

Comment:
I think I'd lose the `dataRateMultipler` and just define the output to be in bytes/s